openwisp-qa-check

```diff
@@ -60,6 +60,16 @@ runcheckendline() {
   fi
 }
 
+runstylelintcheck() {
+  echo "RUNNING: Stylelint checks"
+  stylelint -f verbose $(find . -type f -name "*.css")
+}
+
+runjshintcheck() {
+  echo "RUNNING: jshint checks"
+  jshint --verbose $(find . -type f -name "*.js" -a ! -path "*vendor/*.js")
```

Comment:
Just like above it will give a lot of output even when the test passes.

```
runjshintcheck() {
  jshint $(find . -type f -name "*.js" -a ! -path "*vendor/*.js") &&
    echo "SUCCESS: Stylelint check successful!" ||
    {
      echoerr "ERROR: Stylelint check failed! Hint: did you forget running openwisp-qa-format?"
      FAILURE=1
    }
}
```